### PR TITLE
feat: allow configuring mob start commit message

### LINKED ISSUE #363: Empty commit message for skip CI is hardcoded - breaks 'mob start' depending on push rule
Hi all,

First of all thanks for the nice tool! I really enjoy using it for remote mob programming sessions.

With [4.2.0](https://github.com/remotemobprogramming/mob/releases/tag/v4.2.0), #353 has been released. On start of a mob session, this creates an empty commit to skip CI for the new branch.
Unfortunately, this breaks scenarios where certain [push rules](https://docs.gitlab.com/ee/user/project/repository/push_rules.html#validate-commit-messages) are enforced due to the fact that the commit message is hardcoded here: https://github.com/remotemobprogramming/mob/blob/fcd1f6330ae537ea9942aa98eb1e66702bcca736/configuration/configuration.go#L18

Due to this, we can not upgrade to 4.2.0 - or am I missing a configuration option to either disable this (not preferred - skipping CI makes sense here imho) or change the commit message?

I'd like to suggest aligning this with the implementation of the configuration option 'MOB_WIP_COMMIT_MESSAGE': https://github.com/remotemobprogramming/mob/blob/fcd1f6330ae537ea9942aa98eb1e66702bcca736/configuration/configuration.go#L414

Thanks!

### PR DESCRIPTION
**Pull request by lucas-dclrcq**
Closes #363

### CODE REVIEW

File: configuration/configuration_test.go [hunk 1]
```diff
@@ -219,6 +220,7 @@ func TestReadConfigurationFromFileOverrideEverything(t *testing.T) {
 	test.Equals(t, "Mona", actualConfiguration.TimerUser)
 	test.Equals(t, "https://timer.innoq.io/", actualConfiguration.TimerUrl)
 	test.Equals(t, "team-stash-name", actualConfiguration.StashName)
+	test.Equals(t, "team-stash-name", actualConfiguration.StashName)
```
Comment (hollesse) [inline configuration/configuration_test.go:223]: Why is this duplicated here?

Comment (hollesse) [reply configuration/configuration_test.go:223]: I think you forgot to check for the `actualConfiguration.StartCISkipCommitMessage`

**Review by hollesse — CHANGES_REQUESTED** [review]
Except for the test, where you forget to adjust the duplicated assertion this looks fine to me.

File: configuration/configuration_test.go (hunk 1 above)
Comment (lucas-dclrcq) [reply configuration/configuration_test.go:223]: Yep, my bad I'll fix that right away.

**Review by hollesse — APPROVED** [review]
Thank you for your contribution! The pull request looks good to me now :)

**Pull request merged**